src/tezos/gas.ml

```diff
@@ -0,0 +1,8 @@
+let () = assert (Sys.int_size = 63)
+
+(* TODO: properly implement this, saturation arithmetic *)
```

Comment:
I know Tezos has a whole saturation arithmetic module. Without copying this, can someone do something unsafe?

Reply:
nope, it's still limited to (2 ** 62 - 1) because we don't provide the APIs for it, but yes if we added `( + )` here it would be unsafe.